devops/deploy-on-ec2.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: deploy RRE on EC2
+  hosts: all
+  remote_user: ubuntu
+  become_user: ubuntu
+  tasks:
+    - name: Add nodejs apt key
+      become: yes
+      become_user: root
+      apt_key:
+        url: https://deb.nodesource.com/gpgkey/nodesource.gpg.key
+        state: present
+
+    - name: Add nodejs 16.x ppa for apt repo
+      become: yes
+      become_user: root
+      apt_repository:
+        repo: deb https://deb.nodesource.com/node_16.x focal main
+        update_cache: yes
+
+    - name: Install nodejs
+      become: yes
+      become_user: root
+      ansible.builtin.apt:
+        name: nodejs
+
+    - name: get latest github repo
+      git:
+        repo: https://github.com/transitmatters/regional-rail-explorer.git
+        dest: /home/ubuntu/regional-rail-explorer
+        force: yes
+
+    - name: copy systemd service file
+      become: yes
+      become_user: root
+      copy:
+        src: systemd.conf
```

Comment:
again, somehow it knows we're in the devops folder?

Reply:
yeah, it's local to the playbook